src/PSRule/Configuration/OutputOption.cs

```diff
@@ -168,6 +175,12 @@ internal static OutputOption Combine(OutputOption o1, OutputOption o2)
         [DefaultValue(null)]
         public OutputStyle? Style { get; set; }
 
+        /// <summary>
+        /// The indentation for JSON output
+        /// </summary>
+        [DefaultValue(null)]
+        public int? JsonIndent { get; set; }
```

Comment:
Hmm do you think nullable is needed? Could just use `int` and `[DefaultValue(0)]`.

Reply:
Use nullable because you can't distinguish between **not set** and set with the default value. This is important because when serializing we want to emit an option that is set. Otherwise even if the option is set it will be ignored and not emitted.

Reply:
Good point. I'll leave it as it 🙂